client/src/api/common/powerModel.ts

```diff
@@ -39,9 +45,22 @@ export function usePowerModel() {
   const maxHandler = useCallback((data: EstimatedData) => {
     setEstData(data);
   }, []);
-  useChannelShaped('boost/predicted_max_speed', EstimatedData, maxHandler);
+  useChannelShaped(
+    'power-model-predicted-max-speed',
+    EstimatedData,
+    maxHandler,
+  );
+
+  const handleAchivedMaxData = useCallback((data: AchievedData) => {
+    setAchievedData(data);
+  }, []);
+  useChannelShaped(
+    'power-model-achieved-max-speed',
+    AchievedData,
+    handleAchivedMaxData,
+  );
```

Comment:
Spelling `handleAchivedMaxData -> handleAchievedMaxData`.
There's also a few [spell check extensions](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) if you wish to use them.

Reply:
woah thanks for that :))